### PR TITLE
chore(digitaltwins-device): cleaning up sample_device.js and .ts

### PR DESCRIPTION
**Pull request by YoDaMa**

### CODE REVIEW

File: digitaltwins/samples/device/javascript/sample_device.js [hunk 1]
```diff
@@ -23,19 +31,19 @@ const propertyUpdateHandler = async (component, propertyName, reportedValue, des
...
 const environmentalSensor = new EnvironmentalSensor('environmentalSensor', propertyUpdateHandler, commandHandler);
 const deviceInformation = new DeviceInformation('deviceInformation');
 
-const capabilityModel = 'dtmi:azureiot:samplemodel;1';
+const rootInterfaceId = 'dtmi:contoso_device_corp:samplemodel;1';
 
 async function main() {
   // mqtt is implied in this static method
-  digitalTwinClient = DigitalTwinClient.fromConnectionString(capabilityModel, process.env.DEVICE_CONNECTION_STRING);
+  digitalTwinClient = DigitalTwinClient.fromConnectionString(rootInterfaceId, process.env.DEVICE_CONNECTION_STRING);
```
Comment (rido-min) [inline digitaltwins/samples/device/javascript/sample_device.js:46]: We should avoid the term `Root interface` and use `ModelId` instead.

Comment (YoDaMa) [reply digitaltwins/samples/device/javascript/sample_device.js:46]: @rido-min so are we using Root Interface Id or Device Capability Model?

Comment (rido-min) [reply digitaltwins/samples/device/javascript/sample_device.js:46]: Device Capability Model -> Model (implemented by different interfaces)
DCM Id -> ModelId

There is no such concept as Root Interface in DTDL v2.

/cc @ericmitt @jspaith @jebrando

Comment (YoDaMa) [reply digitaltwins/samples/device/javascript/sample_device.js:46]: this is the PR I am referencing: https://github.com/Azure/azure-iot-sdk-c/pull/1508

Comment (YoDaMa) [reply digitaltwins/samples/device/javascript/sample_device.js:46]: @rido-min can you send @anthonyvercolano and I and email about this new spec, we were unaware of this change...

**Comment by anthonyvercolano** [comment]
>     super(name, 'dtmi:azureiot:DeviceManagement:DeviceInformation;1', propertyCallback, commandCallback);

Charge.

---
Refers to: digitaltwins/samples/device/javascript/deviceInformationInterface.js:11 in e4b8e5e. [](commit_id = e4b8e5e569b52ac4cf3d3f5527aeaa248b7da7ed, deletion_comment = False)

**Comment by anthonyvercolano** [comment]
>     super(componentName, 'dtmi:azureiot:DeviceInformation;1');

change.

---
Refers to: digitaltwins/samples/device/typescript/src/deviceInformationInterface.ts:22 in e4b8e5e. [](commit_id = e4b8e5e569b52ac4cf3d3f5527aeaa248b7da7ed, deletion_comment = False)

File: digitaltwins/samples/device/javascript/sample_device.js [hunk 2]
```diff
@@ -23,19 +31,19 @@ const propertyUpdateHandler = async (component, propertyName, reportedValue, des
 
 const commandHandler = (request, response) => {
```
Comment (rido-min) [inline digitaltwins/samples/device/javascript/sample_device.js:32]: @YoDaMa this statement throws a runtime exception, looks you need to mark the handler as async.

```
/workspace/azure-iot-sdk-node/digitaltwins/samples/device/javascript/sample_device.js:26
  await response.acknowledge(200, 'helpful response text');
  ^^^^^

SyntaxError: await is only valid in async function
```

File: digitaltwins/e2e/dtdl/test_capability_model.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
-  "@id": "urn:microsoft:azureiot:nodesdk:e2etest:cm:1",
+  "@id": "urn:microsoft:contoso_device_corp:nodesdk:e2etest:cm:1",
```
Comment (anthonyvercolano) [inline digitaltwins/e2e/dtdl/test_capability_model.json:2]: >urn [](start = 10, length = 3)

Should these be dtmi?

File: digitaltwins/e2e/dtdl/test_interface.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "@id": "urn:microsoft:azureiot:nodesdk:e2etest:1",
+  "@id": "urn:microsoft:contoso_device_corp:nodesdk:e2etest:1",
```
Comment (anthonyvercolano) [inline digitaltwins/e2e/dtdl/test_interface.json:2]: >urn [](start = 10, length = 3)

dtmi?

File: digitaltwins/samples/device/typescript/src/modelDefinition.ts [hunk 5]
```diff
@@ -7,7 +7,7 @@ import { BaseInterface, Command, PropertyChangedCallback, CommandCallback } from
 export class ModelDefinition extends BaseInterface {
   getModelDefinition: Command;
```
Comment (anthonyvercolano) [inline digitaltwins/samples/device/typescript/src/modelDefinition.ts:8]: I think this whole file goes away.

**Comment by anthonyvercolano** [comment]
> 

Get rid of the .catch and put a ';' at the end of the line.

---
Refers to: digitaltwins/samples/device/typescript/src/sample_device.ts:120 in 0e3c2cd. [](commit_id = 0e3c2cdcd718049eb0e7889121e3546db1bb65ad, deletion_comment = False)

**Pull request merged**